packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -17,6 +17,8 @@ import { RunnableModule } from '@cardano-sdk/util';
 import { toGenesisParams, toLedgerTip, toProtocolParams, toSupply } from './mappers';
 import memoize from 'lodash/memoize';
 
+export const LEDGER_TIP_TTL = 3; // In seconds
```

Comment:
We can use the type we have instead of comment. Also since the tip query is now quite efficient let's go with 1s.
```suggestion
export const LEDGER_TIP_TTL = Seconds(1);
```

Reply:
We could consider setting the cache value based on the slot length